docs/adr/XCall_IBC_Connection.md

```diff
@@ -337,20 +379,32 @@ public void transferAdmin(Address admin) {
 ```
 
 ```java
-public void configureConnection(String connectionId, String portId, String counterpartyNid) {
+/**
+ * Configures a ibc connection so that a channel can be establised for a specigic nid
+
+
+ * @param connectionId The connection id of the connection/chain to open the connection to
+ * @param portId  The allocated port name of the connection on the counterparty chain
+ * @param counterpartyNid The network Id to be associated with this connection
+ * @param clientId The lightClient associated with this connection
+ * @param timeoutHeight The timeoutheight to be used on packets, it is recommended to use a high value similar to the trusting period of the lightclient.
+ */
+public void configureConnection(String connectionId, String portId, String counterpartyNid, String clientId, BigInteger timeoutHeight) {
```

Comment:
Can we change the name of portId to counterpartyPortId?

Reply:
Yeah that would be more clear